api_app/schemas/workspace.json

```diff
@@ -6,8 +6,7 @@
   "description": "These parameters are required for all workspaces",
   "required": [
     "display_name",
-    "description",
-    "address_space_size"
```

Comment:
Don't we want a size prop in every workspace request?

Reply:
It's set in the required section of the workspace specific `tempalte_schema.json`, which is where the `address_space_size` property also sits. Having it here means if there is no address space size in the workspace specific JSON you get an error. 

I can put this back, as doesn't directly affect this PR, but the required should be in the same file as the property itself?

Reply:
Also it defaults to small within the api if not included in the request.